src/Http/Controllers/OpenApiJsonController.php

```diff
@@ -22,7 +22,7 @@ protected function getJson() : array
     {
         $path = config('swagger-ui.file');
 
-        if (Str::endsWith($path, '.yaml')) {
+        if (!filter_var($path, FILTER_VALIDATE_URL) && Str::endsWith($path, '.yaml')) {
```

Comment:
I don't think this is necessary. If the file is an url, but the file behind that url is a yaml file, then we should still be able to parse the file.

The only thing we probably have to to is change `yaml_parse_file($path);` to `yaml_parse(file_get_contents($path))`